crt_portal/static/sass/custom/error.scss

```diff
@@ -0,0 +1,42 @@
+.crt-error-box {
+  display: flex;
+  align-items: flex-start;
+  padding: 1rem;
+
+  span {
+    padding: 0 .5rem;
+
+    li {
+      list-style: none;
+    }
+  }
+}
+
+.error-subheading {
+  margin: 0;
+  font-weight: $theme-font-weight-bold;
+  color: color($theme-color-primary-darker);
+}
+
+.crt-error-card {
```

Comment:
Oh, I missed this but if we are copying from another source, we can do `@extend .crt-portal-card` in SCSS.